packages/@expo/metro-config/src/transform-plugins/__tests__/__snapshots__/import-export-plugin-snapshots.test.ts.snap

```diff
@@ -496,31 +550,30 @@ Object.defineProperty(exports, "_test", {
     }(_appleIcons);
   }
 });
-var _appleIcons6 = require("apple-icons");
-Object.keys(_appleIcons6).forEach(function (_key) {
+var _appleIcons = require('apple-icons');
+Object.keys(_appleIcons).forEach(function (_key) {
   if (_key === "default" || _key === "__esModule") return;
   if (Object.prototype.hasOwnProperty.call(_exportedNames, _key)) return;
-  if (_key in exports && exports[_key] === _appleIcons6[_key]) return;
+  if (_key in exports && exports[_key] === _appleIcons[_key]) return;
   Object.defineProperty(exports, _key, {
     enumerable: true,
     get: function () {
-      return _appleIcons6[_key];
+      return _appleIcons[_key];
     }
   });
 });
-var _appleIcons = require('apple-icons');
-var _appleIcons4 = _$$_IMPORT_ALL('apple-icons');
-var _appleIcons5 = _$$_IMPORT_ALL('apple-icons');
-exports.AppleIcons = _appleIcons4;
-exports.default = _appleIcons5;"
+var _appleIcons2 = _$$_IMPORT_ALL('apple-icons');
+var _appleIcons3 = _$$_IMPORT_ALL('apple-icons');
```

Comment:
technically rewritable as:

```js
var sharedRequire = require('...');
var namespaceConversion = sharedRequire && sharedRequire.__esModule ? sharedRequire : {
  ...sharedRequire, default: sharedRequire.default,
};
```

Based on https://github.com/facebook/metro/blob/460eca02e2c804d42633b0d92e61da87cad3cf80/packages/metro-runtime/src/polyfills/require.js#L294-L305

This would allow us to further simplify some state